src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:first-child,
+.stars--2 > .stars__star:not(:nth-child(1n+3)),
+.stars--3 > .stars__star:not(:nth-child(1n+4)),
+.stars--4 > .stars__star:not(:nth-child(1n+5)),
+.stars--5 > .stars__star:not(:nth-child(1n+6)) {
```

Comment:
use [cycle](https://sass-lang.com/documentation/at-rules/control/for)

Reply:
done